library/Notifications/Common/Icons.php

```diff
@@ -41,4 +41,32 @@ private function __construct()
     public const RULE_MATCHED = 'filter';
 
     public const UNDEFINED = 'notdef';
+
+    public const ACKNOWLEDGED = 'check';
+
+    public const UNACKNOWLEDGED = 'xmark';
+
+    public const DOWNTIME = 'plug';
+
+    public const FLAPPING = 'bolt';
+
+    public const INCIDENT_AGE = 'hourglass-end'; //TODO: icon? and for custom?
+
+    public const SEVERITY_OK = 'heart';
+
+    public const SEVERITY_CRIT = 'circle-exclamation';
+
+    public const SEVERITY_WARN = 'exclamation-triangle';
```

Comment:
On fontawesome.com this is `triangle-exclamation`

Reply:
They changed it on v6, it still works, but I will change it.